### PR TITLE
create_layout

### PR DESCRIPTION
**Pull request by Kushan1994**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -18,3 +18,7 @@
 })->middleware(['auth', 'verified'])->name('dashboard');
 
 require __DIR__.'/auth.php';
+
+Route::get('/test', function (){
+    return Inertia::render('test');
+});
```
Comment (sureshhemal) [inline routes/web.php:24]: create separate controller. 

`HomePageController.php` and call a method named '__invoke'

**Review by sureshhemal — CHANGES_REQUESTED** [review]

File: routes/web.php (hunk 1 above)
Comment (sureshhemal) [reply routes/web.php:24]: `Route::get('/home', HomePageController)->name('home');`

File: app/Http/Controllers/HomePageController.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+public function _invoke() {
+
```
Comment (sureshhemal) [inline app/Http/Controllers/HomePageController.php:2]: `return Inertia::render('test');`

**Review by sureshhemal — CHANGES_REQUESTED** [review]

**Review by sureshhemal — CHANGES_REQUESTED** [review]

File: resources/js/Components/LayoutComponents/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<script setup></script>
```
Comment (sureshhemal) [inline resources/js/Components/LayoutComponents/Footer.vue:1]: remove unnecessary script tags from components

File: resources/js/Components/LayoutComponents/LineFooter.vue [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<script setup></script>
```
Comment (sureshhemal) [inline resources/js/Components/LayoutComponents/LineFooter.vue:1]: remove

File: resources/js/Components/LayoutComponents/Navbar.vue [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<script setup></script>
```
Comment (sureshhemal) [inline resources/js/Components/LayoutComponents/Navbar.vue:1]: remove script tag

File: resources/js/Components/LayoutComponents/Topbar.vue [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<script setup></script>
```
Comment (sureshhemal) [inline resources/js/Components/LayoutComponents/Topbar.vue:1]: remove

File: resources/js/Layouts/Frontend.vue [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    <div class="bg-gray-50">
+        <TopBar />
+
+        <Navbar />
+
+        <main class="overflow-y-auto focus:outline-none">
+               <slot/>
+        </main>
+
+        <Footer />
```
Comment (sureshhemal) [inline resources/js/Layouts/Frontend.vue:18]: As we discussed there is two footers. Which one is this one and where the other? One should be fixed on bottom of the page

**Review by sureshhemal — CHANGES_REQUESTED** [review]

File: resources/js/Layouts/Frontend.vue (hunk 7 above)
Comment (Kushan1994) [reply resources/js/Layouts/Frontend.vue:18]: yes. i did it. inside the line footer components

**Review by sureshhemal — APPROVED** [review]

**Pull request merged**